packer/Dockerfile

```diff
@@ -1,20 +1,33 @@
 # Due to this faccessat2 change: https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2
 # we need to continue to use alpine 3.13 until we upgrade to using Docker 20.10.0 or greater, AND libseccomp 2.4.4
 # or greater on concourse workers.
-FROM alpine:3.13
+FROM ubuntu:20.04
 
 # IF you update the packer version here you should also rebuild the https://github.com/ONSdigital/packer-resource
 ENV PACKER_VERSION=1.7.8
-ENV ANSIBLE_VERSION=2.10.7-r0
+LABEL "packer_version"="1.7.8"
 
-RUN apk add --no-cache \
+
+ENV ANSIBLE_VERSION=2.10.7
+LABEL "ansible_version"="2.10.7"
```

Comment:
Move ENV and LABEL commands closer to where they are needed, see the comments further down